frontend/views/GamesView.tsx

```diff
@@ -5,14 +5,11 @@ import { PageTitle } from '../components/styles/PageTitle';
 import { PageSubtitle } from '../components/styles/PageSubtitle';
 import { PageSection } from '../components/styles/PageSection';
 import { SearchRow } from '../components/styles/SearchRow';
-import { Store } from 'redux';
-import { AppStore, AppActions, BreadcrumbActions } from '../storage/common';
 import GamesList from '../components/GamesList';
 import { Link } from 'react-router-dom';
 
-interface GamesViewProps {
-	store: Store<AppStore, AppActions>;
-}
+//There has to be some kind of empty props, for constructor and state assigment in definition of Component
+interface GamesViewProps {}
```

Comment:
You can replace the usage of this with `{}`, eg:

![image](https://user-images.githubusercontent.com/4552000/76828760-eeede900-6821-11ea-96ba-7c0d7ee0527a.png)

I personally find this clearer than an empty interface.